components/CaseStudyCard.tsx

```diff
@@ -19,11 +19,11 @@ export default function CaseStudyCard({ studies = [] }: ICaseStudyCardProps) {
   }
 
   return (
-    <div className='flex flex-wrap gap-3 pt-10 lg:grid lg:grid-cols-3 lg:gap-8 lg:text-center'>
+    <div className='flex flex-wrap gap-3 pt-10 lg:flex lg:items-center  lg:justify-center lg:gap-8 lg:text-center'>
```

Comment:
Since we are using flex for all viewports, can't we simplify this to use flex in all viewports?

@CodeRabbitai can you suggest something here?

Reply:
@cbum-dev Can you please make this change?